src/python/dxpy/workflow_builder.py

```diff
@@ -453,7 +454,7 @@ def _build_underlying_workflows(enabled_regions, json_spec, args):
     try:
         for region, project in projects_by_region.items():
             json_spec['project'] = project
-            workflow_id = _build_regular_workflow(json_spec)
+            workflow_id = _build_regular_workflow(json_spec, args.keep_open)
```

Comment:
Here we don't have to pass args.keep_open since we require that the underlying workflows in the global workflow are closed so the arg will always be false.

Reply:
fixed